doc/api/deprecations.md

```diff
@@ -582,6 +582,16 @@ deprecated.
 *Note*: `OutgoingMessage.prototype._renderHeaders` was never documented as
 an officially supported API.
 
+<a id="DEP0068"></a>
+### DEP0068: vm.runInDebugContext(string)
+
+Type: Runtime
+
+The DebugContext is being removed from V8 5.8+ along with the legacy debugger
+interface.
```

Comment:
is it possible to mention when v8 5.8 is expected to land? Most people don't know what version of v8 is in specific node versions, so I suggest adding something like "Deprecated to warn users that this function will have to be deleted when Node.js next updates V8." (or something of the like).